tests/compiler/test_opcodes.py

```diff
@@ -39,7 +39,7 @@ def test_version_check(evm_version):
     assert opcodes.version_check(begin=evm_version)
     assert opcodes.version_check(end=evm_version)
     assert opcodes.version_check(begin=evm_version, end=evm_version)
-    if evm_version != "byzantium":
+    if evm_version != "byzantium" and evm_version != "atlantis":
```

Comment:
I think we should perhaps consider enumerating these values ?

```
class EVMVersion:
     ATLANTIS = "atlantis"
```


Also I believe `evm_version in {"byzantium", "atlantis"}` is easier to read than two and statements.

Reply:
```suggestion
    if evm_version in ("byzantium", "atlantis"):
```